source/frontend/src/components/common/form/UserRegionSelect/UserRegionSelectContainer.tsx

```diff
@@ -26,10 +27,10 @@ export const UserRegionSelectContainer: React.FunctionComponent<
   );
   const { retrieveUserInfo, retrieveUserInfoResponse } = useUserInfoRepository();
   const regionTypes = getOptionsByType(API.REGION_TYPES);
-  const userRegionCodes = retrieveUserInfoResponse?.userRegions?.map(ur =>
-    ur.regionCode?.toString(),
-  );
-  const userRegionTypes = regionTypes.filter(r => userRegionCodes?.includes(r.code));
+  const userRegionCodes =
```

Comment:
hmm, not so sure about this one. doesn't seem like a non-functional change. seems fine overall but the r.code ?? '' seems odd.

Reply:
I agree. I left it like that to maintain curent functionalty. Otherwise, filtering valid codes could cause a sideeffect